plugins/mysql_helpers.sh

```diff
@@ -5,7 +5,7 @@
 # Author: Matteo Cerutti <matteo.cerutti@hotmail.co.uk>
 #
 
-MYCNF_PATH="$ZBX_HOME/.my.cnf"
+MYCNF_PATH="/usr/share/zabbix/.my.cnf"
```

Comment:
This might break what ppl have set up in their environments. Can you perhaps do the following:

```shell
ZBX_HOME=${ZBX_HOME:-/usr/share/zabbix}
```